cmd/access/node_builder/access_node_builder.go

```diff
@@ -71,8 +71,8 @@ type AccessNodeBuilder interface {
 	// IsStaked returns True is this is a staked Access Node, False otherwise
 	IsStaked() bool
 
-	// ParticipatesInUnstakedNetwork returns True if this is a staked Access node which also participates
-	// in the unstaked network acting as an upstream for other unstaked access nodes, False otherwise.
+	// ParticipatesInUnstakedNetwork returns True if this an Access Node which participates in the unstaked network,
```

Comment:
since there is no separate `unstaked network` now we can call this `SupportsConsensusFollower`.

Reply:
Yes, I agree